components/toggleButton/toggleButton.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { getClassNamesWithMods } from '../_helpers';
+
+/**
+ * ToggleButton component.
+ */
+class ToggleButton extends Component {
+  /**
+   * Initialize default state
+   */
+  state = {
```

Comment:
I would prefer to make it stateless and control selected item from outside. With current implementation, it is not possible to pre select item by default